docs/consume-packages/Package-References-in-Project-Files.md

```diff
@@ -247,9 +249,9 @@ If `ProjectA` has a dependency on a `PackageX` version `2.0.0` and also referenc
 
 You can control various behaviors of restore with lock file as described below:
 
-| Option | MSBuild equivalent option | Description|
-|:---  |:--- |:--- |
-| `--use-lock-file` | RestorePackagesWithLockFile | Opts into the usage of a lock file. | 
-| `--locked-mode` | RestoreLockedMode | Enables locked mode for restore. This is useful in CI/CD scenarios where you want repeatable builds.|   
-| `--force-evaluate` | RestoreForceEvaluate | This option is useful with packages with floating version defined in the project. By default, NuGet restore will not update the package version automatically upon each restore unless you run restore with this option. |
-| `--lock-file-path` | NuGetLockFilePath | Defines a custom lock file location for a project. By default, NuGet supports `packages.lock.json` at the root directory. If you have multiple projects in the same directory, NuGet supports project specific lock file `packages.<project_name>.lock.json` |
+| NuGet.exe option | dotnet pption | MSBuild equivalent option | Description |
+|:--- |:--- |:--- |:--- |
+| `-UseLockFile` |`--use-lock-file` | RestorePackagesWithLockFile | Opts into the usage of a lock file. |
+| `-LockedMode` | `--locked-mode` | RestoreLockedMode | Enables locked mode for restore. This is useful in CI/CD scenarios where you want repeatable builds.|   
+| `-ForceEvaluate` | `--force-evaluate` | RestoreForceEvaluate | This option is useful with packages with floating version defined in the project. By default, NuGet restore will not update the package version automatically upon each restore unless you run restore with this option. |
+| `-LockFilePath` | `--lock-file-path` | NuGetLockFilePath | Defines a custom lock file location for a project. By default, NuGet supports `packages.lock.json` at the root directory. If you have multiple projects in the same directory, NuGet supports project specific lock file `packages.<project_name>.lock.json` |
```

Comment:
Would it be better if this table was sorted?

Reply:
I think we should close in favor of https://github.com/NuGet/docs.microsoft.com-nuget/pull/1921...I know it's later, but I am actively working on it, so it stands a better chance at working.

Reply:
I thought Iw as replying on a different issue :D
I think they are ordered in importance order, I think that's ok.